spec/fpm/package/rpm_spec.rb

```diff
@@ -39,6 +39,19 @@
     end
   end
 
+  describe "#summary" do
+    it "should default to description" do
+      expected = subject.description
+      insist { subject.summary } == expected
+    end
+
+    it "should return description override" do
+      subject.attributes[:rpm_summary] = "a summary"
+      expected = subject.description
+      insist { subject.summary } != expected
```

Comment:
I've changed it to subject.attributes[:rpm_summary] but setting the attribute alters the summary in the subject anyway since subject.summary is a method that returns the summary and not the @summary class attribute I think? If you could confirm that'd be good.